docs/operating-scylla/procedures/cluster-management/replace-running-node.rst

```diff
@@ -16,9 +16,14 @@ Add a new node to the cluster and then decommission the old node
 =================================================================
 
 Adding a new node and only then decommissioning the old node allows the cluster to keep the same level of data replication throughout the process, but at the cost of more data being transferred during the procedure.
-When adding a new node to a Scylla cluster, existing nodes will give the new node responsibility for a subset of their vNodes, making sure that data is once again equally distributed. In the process, these nodes will stream relevant data to the new node.
-When decommissioning a node from a Scylla cluster, it will give its vNodes to other nodes, making sure data is once again equally distributed. In the process, this node will stream its data to the other nodes.
-Hence, replacing a node by adding and decommissioning redistribute the vNodes twice, streaming a node worth of data each time.
+When adding a new node to a ScyllaDB cluster, it will take up ownership of a subset of existing nodes their data.
```

Comment:
The phrase "their data" doesn't seem to fit this sentence. I'm not sure what you meant to say here.

By the way, I didn't understand what wrong in the old text here? What was wrong?
By the way, both pre-patch and post-patch focus on vnodes, and it probably doesn't happen the same way with tablets.